GPU/GPUCommonHW.cpp

```diff
@@ -819,6 +820,7 @@ void GPUCommonHW::FastRunLoop(DisplayList &list) {
 			if (flags & (FLAG_EXECUTE | FLAG_EXECUTEONCHANGE)) {
 				downcount = dc;
 				(this->*info.func)(op, diff);
+				// TODO: Check pc here, and break if invalid, as the func can change it. Might have a performance impact though.
```

Comment:
I'd rather check it when modifying the PC, rather than after every func (which only a small % of them can change the PC.)

-[Unknown]